pkg/sfu/videolayerselector.go

```diff
@@ -1,202 +0,0 @@
-package sfu
```

Comment:
moving this to `videolayerselector` directory and making it satisfy the `VideoLayerSelector` interface